app/models/blacklight_config.rb

```diff
@@ -0,0 +1,27 @@
+class BlacklightConfig
+  attr_reader :metadata_class
+  def initialize(metadata_class)
+    @metadata_class = metadata_class
+    apply_show_fields
+  end
+
+  def default_configuration
+    CatalogController.blacklight_config.deep_copy
```

Comment:
Could we pass this in rather than having a model that directly talks to a controller?